### DISCUSSION TITLE
Referral Program

### DISCUSSION THREAD

**Discussion by NilzGone** [sunflower-land/sunflower-land#2402] [Ideas]
Hi, I think the referral program is good but I don't like the idea of getting $1 of my friend's money, I get it was going to be spent on the farm anyway but it feels weird.

I don't have the best idea of what could be better but I wanted to open this discussion to see what other people though.

The could ideas I do have - 

The referred account could get an xp boost for the first month of playing (maybe the referrer as well but it would probably cap at 1 boost)

Some kind of crop growing aim, e.g. if 10000 sunflowers are grown between the linked accounts within a month you get some kind of cosmetic or maybe a small boosting NFT.

Maybe even the option of paying for the farm for the new account, I'm much more likely to introduce people I actually know if it was free for them (even if it was paid by me). This probably doesn't work for referring people online like through Twitter etc but maybe the option for it as well? like a pre-minted farm

**Comment by adamhannigan** [comment]
Definitely, we want to add some in-game rewards + boosts for the referral program, we just haven't built them yet. ❤️

**Reply by ghost** [reply]
I think you can think about a system of friends, which will also be associated with a referral system 

I had a few ideas about this, but they still need a lot of fine-tuning that would not be abused by multiple farm owners. https://github.com/sunflower-land/sunflower-land/discussions/2317

**Reply by future2back** [reply]
@adamhannigan 

I believe that limiting a group to be eligible for receiving an NFT is always a bad idea, so I don't agree with the idea to give boosts NFTs to marketing personnel, **NFTs should be for players!**


In my opinion, the referral system is unappealing and seems desperate. But if you wanna try, what advantage does a player gain by joining the game through a referral link? You also could consider other ideas besides giving $1 for each referral, such as implementing a system that offers rewards based on rankings per season. 

For example:

```
"Remaining XXX players until the end of season."

1st - $1500 
2nd - $1000
3rd - $500
4th to 10th - $250
11th to 20th - $100
21th to 100th - $10
```

One possible way to utilize the funds collected is by investing $2 in real and effective marketing tools, such as Google and Meta, and using the other $1 to distribute as prizes to players on the leaderboard (Total $3). If making the rewards in dollars, as our friend @NilzGone suggested, seems weird, we could consider using the dollars to purchase SFL and then proceed with rewards in SFL. 


If the game's entry fee is $5, the divided budget could be allocated in such a way as to maintain greater sustainability, while also providing opportunities for artists community to contribute to marketing campaigns.

![image](https://user-images.githubusercontent.com/74167741/230706210-aae9a4e0-631d-4f8a-822d-c54192ce5b74.png)





In order to keep players engaged and actively playing, we urgently need to modify the monotony of the gameplay in this game. We need more entertainment and interactivity among players. The real-time multiplayer map should be a priority, as it can bring new possibilities to the game. These possibilities include resource burning and, more importantly, halting the generation of new resources.

If our server can already accommodate 500 players, imagine how exciting it would be to have 5 guilds (with 100 players each) participating in a tournament every 15 days, after spending a significant amount of resources to register for the event. 
The winners would gain access to an exclusive co-op map with special benefits. 
500 players online, actively playing the game for 1 hour, burning resources to help the economy and enjoying the competition, it would incentivize players to strive for greater strength and desire items with buffs, and so on. However, we must not forget that it needs to be fun, hence the real-time online multiplayer.

We need to bring new experiences to the players.


Why don't we discuss this with the community?

**Reply by offerT** [reply]
It is necessary to accelerate the participation of new users
